viewer/js/gis/dijit/LayerControl/plugins/legendUtil.js

```diff
@@ -233,10 +233,16 @@ define([
         // create legends for vector layers (feature, etc)
         _createVectorLegend: function (infos, layer, expandNode) {
             //create legend table
-            var table = domConst.create('table');
+            var table = domConst.create('table'),
```

Comment:
The purpose of this change is to handle behavior of certain feature layers in the layer control's legend (AKA TOC). There are certain layers, including some used for WATERSS, which is how we came to discover this here, that show up with multiple, repeated labels.